lib/taurus/core/util/log.py

```diff
@@ -793,6 +793,15 @@ def _format_stack(self, stack_func=inspect.stack):
                         out += '<could not find suitable string representation>'
         return out
 
+    def loggingtofile(self, msg, logging_path, *args, **kw):
```

Comment:
Not problem in changing the name. I would have to change it after that in the Sardana PR.